articles/agora-express-token-server/index.md

```diff
@@ -0,0 +1,408 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/agora-express-token-server/
+title: Express Server to Generate Authentication Token for Agora Applications
+description: This tutorial will give readers a detailed guide on how they can build an app to create or join a livestream using Agora.
```

Comment:
Hey @hectorkambow, The description in the front matter is inaccurate. Can I update it?

Reply:
This tutorial will give readers a detailed guide on how to implement authentication for applications using Agora's RTC SDK or RTM SDK.

Reply:
This tutorial will give readers a detailed guide on how they can implement authentication for the Agora RTC SDK and RTM SDK.

Reply:
I updated it.

Reply:
That would be why I got a merge conflict 🤔

Reply:
uh-oh

Reply:
> This tutorial will give readers a detailed guide on how they can implement authentication for the Agora RTC SDK and RTM SDK.

Please try to include this description in the beginning - (150 characters) can be include in the PR notes.

 Trying to fix this merge conflict.

Reply:
hmm, I don't see a merge conflict. 🤔

Reply:
Its on my end (I was in the middle of adding my changes) - could you please re-submit the PR with the images files. 

I'll fix it in the new PR. 
👍  
Thank you 🚀